pandas/io/html.py

```diff
@@ -1058,6 +1106,11 @@ def read_html(
     displayed_only : bool, default True
         Whether elements with "display: none" should be parsed.
 
+    extract_hrefs : all/header/body/footer or None, default None
```

Comment:
these need to be a list of strings (e.g. the formatting),

Reply:
Better now?